modules/jaeger-install-elasticsearch.adoc

```diff
@@ -20,7 +20,7 @@ Do not install Community versions of the Operators. Community Operators are not
 
 [NOTE]
 ====
-If you have already installed the Elasticsearch Operator as part of OpenShift Logging, you do not need to install the Elasticsearch Operator again.  The Jaeger Operator will create the Elasticsearch instance using the installed Elasticsearch Operator.
+If you have already installed the OpenShift Elasticsearch Operator as part of OpenShift Logging, you do not need to install the OpenShift Elasticsearch Operator again.  The Jaeger Operator will create the Elasticsearch instance using the installed OpenShift Elasticsearch Operator.
```

Comment:
`.  ` here as well

Reply:
Replaced this one plus three others in this file.